SigningService/IPackagePartSigner.cs

```diff
@@ -1,9 +1,13 @@
+using System.IO;
```

Comment:
This should not exist in the root project directory. Lets move it into \Signers\